Solutions/Alsid For AD/Analytic Rules/PasswordSpraying.yaml

```diff
@@ -11,13 +11,21 @@ queryFrequency: 2h
 queryPeriod: 2h
 triggerOperator: gt
 triggerThreshold: 0
-status: "Available"
+status: Available1
```

Comment:
Hi @rahul0216, seems like a typing error, can you please correct `status` as Available?

Reply:
Thanks @v-rbajaj for pointing out. Corrected the typo.